packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -578,6 +578,11 @@ describe('e2e', function () {
       shell.assertNoErrors();
     });
 
+    it('runs a custom log command', async function () {
+      await shell.executeLine("log.info('Try me')");
+      shell.assertNoErrors();
+    });
```

Comment:
I was about to comment that this should assert that the message is actually logged, but then it seems a test further below already checks this. Does this test do anything that the other doesn't? The `shell.assertNoErrors()` call can and should be added to the other tests as well.